src/components/UserPreferencesPanel/PreferencesCategories.tsx

```diff
@@ -33,15 +55,15 @@ export default function PreferencesCategories() {
         css={css`
           display: grid;
           gap: 1rem;
-          grid-template-columns: 2fr 1fr 1fr 1fr;
+          grid-template-columns: 2fr ${' 1fr'.repeat(channels.length)};
```

Comment:
We might want to add a `trim` here to remove the double space between `2fr` and the first `1fr`.

```suggestion
          grid-template-columns: 2fr ${' 1fr'.repeat(channels.length).trim()};
```